src/actions_manager.coffee

```diff
@@ -89,4 +89,15 @@ define [
 
       payload
 
+    _buildMetadata: (sbm) ->
+      return (@session.metadata || '') unless sbm
+
+      if @session.metadata
+        @session.metadata.tags = if @session.metadata.tags then tags + ',sbm' else 'sbm'
```

Comment:
Probably I m missing something

but 'tags +' is going to be defined somehow?

Reply:
oops, sorry missed that after pushing the latest fixup
re-fixing it now